src/models/facility/index.ts

```diff
@@ -0,0 +1,36 @@
+interface Facility {
+  id: string;
+  data: FacilityData;
+}
+
+export interface FacilityData {
+  name: string;
+  latlng: {
+    latitude: number;
+    longitude: number;
+  };
+  target: {
+    name: string;
+    description: string;
+  };
+}
+
+export type Facilities = Record<string, FacilityData>;
+
+// for origin json data
+type DataFromJsonItem = {
+  region: string;
+  locationName: string;
+  coordinates: {
+    latitude: number;
+    longitude: number;
+  };
+  category: {
+    categoryName: string;
+    categoryDescName: string;
+  };
+};
+
+export type DataFromJson = DataFromJsonItem[];
```

Comment:
```suggestion
export type rawFacilities = rawFacilityData[];
```
Please also apply this change to every other parts of this PR.

Reply:
Remove DataFromJson Type